README.md

```diff
@@ -32,6 +32,10 @@ Have questions? Feedback? Jump in slack and hang out with us
 
 https://join.slack.com/t/trufflehog-community/shared_invite/zt-pw2qbi43-Aa86hkiimstfdKH9UCpPzQ
 
+and check out the Secret Scanning Discord 
+
+https://discord.gg/zpB2jDhr
```

Comment:
Very nit, but it would be nice to have these as formatted links

```
Join our [Slack Community](https://join.slack.com/....)
Join the [Secret Scanning Discord](https://discord.gg/...)
```

Reply:
I like what you're sayin', just updated